Doc/changes.txt

```diff
@@ -1,4 +1,104 @@
-TTX/FontTools Version 2.4
+## TTX/FontTools Version 3.0
+
+2015-09-01 <https://github.com/behdad/fonttools/commit/58f86f318af8b2f8b553e8cceef678d6ce0017c5>
+
+- [Snippets] Add script for cmap subtable format conversion
+- [Snippets] Add `layout-features.py` that prints GSUB/GPOS features
+- [feaLib] Add lexer for OpenType feature files
+- [ttx] add support for dumping WOFF2 fonts to TTX
```

Comment:
Yep, I hate github wikis, much prefer markdown files in the github repo
(and moving from master to gh-pages, so they can be accessed online nicely
:)